src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -1046,9 +1058,17 @@ def __init__(self, config):
             self.projection = nn.Conv2d(256, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
 
         features = config.fusion_hidden_size
+
+        image_size = (
+            config.backbone_config.image_size
+            if config.backbone_config is not None and not config.is_hybrid
+            else config.image_size
+        )
+        modifier = {"size": (int(image_size), int(image_size))} if config.use_size else {"scale_factor": 2}
```

Comment:
This isn't robust as sometimes `image_size` can be a tuple in the model configs.